.github/workflows/js-ci.yml

```diff
@@ -194,13 +194,21 @@ jobs:
     strategy:
       matrix:
         container: [1, 2, 3, 4, 5]
+        browser: [chrome, firefox]
     steps:
       - uses: actions/checkout@v3
 
       - name: Install Google Chrome
         uses: browser-actions/setup-chrome@v1
         with:
           chrome-version: stable
+        if: matrix.browser == 'chrome'
```

Comment:
Just to keep things in line with other workflows stylistically, can we put the `if` after `name`? It makes it easier at a glance to see if a step is executed, rather than having to read all the way down.